src/filterScreenings.js

```diff
@@ -0,0 +1,71 @@
+/*Filters out ten screenings in the next five days or removes the last day's screenings incase
+the number of screenings pushes the total number of screenings to more than ten*/
+export function filterScreenings(data) {
+  let filteredScreenings = [];
+  data.forEach((screening) => {
+    
+    // Setting a starting data for the next day to start displaying screenings
+    const currentScreeningDate = new Date(screening.attributes.start_time);
+    const startingPoint = new Date(Date.now() + 86400000);
+    startingPoint.setHours(0);
+    startingPoint.setMinutes(0);
+    startingPoint.setSeconds(0);
+    startingPoint.setMilliseconds(0);
```

Comment:
What's your reasoning for not including today's screenings?
Also, the `setHours` can take in minutes, seconds and ms as well, so the three following lines could be removed by changing it to `setHours(0,0,0,0)`.
[Ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours)